packages/cozy-harvest-lib/src/components/__snapshots__/OAuthForm.spec.js.snap

```diff
@@ -14,9 +14,9 @@ exports[`OAuthForm should handle oauth cancelation 1`] = `
     }
   />
   <DefaultButton
-    busy={true}
+    busy={false}
```

Comment:
@doubleface @taratatach is it really wanted to change this value? Is it because before we didn't wait the update of the component?

Reply:
The test now waits for the fetch of extra params to be done, which explains why the button is not busy anymore. And also, the test has been rewritten since the move of the OAuthForm component to a functionnal component.

I also checked manually and the buton still displays the busy state when loading extra parameters.

Reply:
Is it a good idea to have : 
- first test that check if the button is busy while the hook is / will be called 
- Then resolve the hook then check that the button is not busy anymore 

?

Reply:
Ok I added a test for busy and not busy state